docker-compose.yml

```diff
@@ -2,22 +2,23 @@ version: '2'
 
 networks:
   front:
-    external:
-      name: front
+    external: true
+    name: front
```

Comment:
Is this network still needed? I don't remember having a "front" network in any other docker-compose.

Reply:
You are right, I have deleted it